test/control-plane/node/ciliumnodes/generate.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+#
+# Generate the golden test files for the Node test
+#
+
+set -eux
+
+export KUBECONFIG=kubeconfig
+
+versions=(1.24)
+
+for version in ${versions[*]}; do
+    mkdir -p v${version}
+
+    : Start a kind cluster
+    kind create cluster --config manifests/kind-config-${version}.yaml --name cilium-nodes
+
+    : Install cilium
+    cilium install --wait
```

Comment:
Should we specify Cilium version that is installed here, or do we expect `generate.sh` to be run in a controlled environment with latest `cilium-cli` installed?

Reply:
Given that this script should only be run when we need to regenerate the test data, I _think_ we can assume that it will be executed in an environment with the latest version of `cilium-cli` installed.